src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -4,12 +4,9 @@
 #include "pal_ssl.h"
 #include <dlfcn.h>
 
-// TLS 1.3 is only defined with 10.13 headers, but we build on 10.12
-#define kTLSProtocol13_ForwardDef 10
-
 // 10.13.4 introduced public API but linking would fail on all prior versions.
 // For that reason we use function pointers instead of direct call.
-// This can be revisited after we drop support for 10.12.
+// This can be revisited after we drop support for 10.12 and iOS 10
```

Comment:
I initially tried getting rid of the function pointers, but learned that we target iOS 10 which does not have these. So, clarify in the comment that iOS 10 support is a condition for keeping these around.